nvme.c

```diff
@@ -6922,7 +6920,6 @@ static int dir_send(int argc, char **argv, struct command *cmd, struct plugin *p
 		  OPT_SHRT("dir-spec",       'S', &cfg.dspec,          dspec_w_dtype),
 		  OPT_BYTE("dir-oper",       'O', &cfg.doper,          doper),
 		  OPT_SHRT("endir",          'e', &cfg.endir,          endir),
-		  OPT_FLAG("human-readable", 'H', &cfg.human_readable, human_readable_directive),
```

Comment:
Best leave it the option around, just to avoid to break anyone. Couldn't we map it to verbose mode ?

Reply:
But if there is no user left, just leave the option. You could add a help text like `deprecated; does nothing`